eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -83,7 +85,7 @@ jobs:
             - (Fedora.34.Amd64.Open)ubuntu.1604.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-34-helix-20210728124700-4f64125
       - ${{ if or(eq(parameters.jobParameters.interpreter, 'true'), eq(parameters.jobParameters.isSingleFile, true)) }}:
         # Limiting interp runs as we don't need as much coverage.
-        - Debian.9.Amd64.Open
+        - Debian.10.Amd64.Open
```

Comment:
> .packages/microsoft.dotnet.helix.sdk/6.0.0-beta.21419.2/tools/Microsoft.DotNet.Helix.Sdk.MonoQueue.targets(55,5): error : (NETCORE_ENGINEERING_TELEMETRY=Helix) ArgumentException: Unknown QueueId. Check that authentication is used and user is in correct groups.

@ViktorHofer, singlefile test leg is failing like this ☝️ does it mean `Debian.10.Amd64` queue does not exist?

Reply:
Right, according to https://helix.dot.net/, there are no debian 10 or debian 11 queues. I assume we would need to use docker images. Opened an internal ticket for clarification: https://github.com/dotnet/core-eng/issues/14163.

Reply:
Image reference worked 👍